### PR TITLE
Made python code pep8 compatible.

### PR DESCRIPTION
**Pull request by deisi**
I went through your python code for the nautilus extension. The changes I made are purely about coding style. They should not effect the functionality in any way. 

See for information.
https://www.python.org/dev/peps/pep-0008/

I tested on my arch linux box if I could still cast from nautilus with and without transcoding. Both was working just fine.

### CODE REVIEW

**Comment by deisi** [comment]
I would also like to change the:

https://github.com/Rafostar/gnome-shell-extension-cast-to-tv/blob/e0408d366ec29705588a8507f8da6e9e339b31a5/nautilus/nautilus-cast-to-tv.py#L44-L49

part of the code, as a general `try`...`except` is not good practice in python. Do you know under what circumstances the try fails? I tried to trigger it on my machine but couldn't make it fail. I found https://docs.python.org/3/library/locale.html#locale.Error as error for `setlocale` but I don't see how `gettext.bindtextdomain` and `gettext.textdomain` fails.

File: nautilus/nautilus-cast-to-tv.py [hunk 1]
```diff
@@ -1,11 +1,16 @@
 # Cast to TV Nautilus/Nemo Extension
 # Developers: Rafostar, rendyanthony
 
-import os, sys, json, gettext, locale, gi
+import os
+import sys
+import json
+import gettext
+import locale
+import gi
```
Comment (Rafostar) [inline nautilus/nautilus-cast-to-tv.py]: I prefer having those imports in a single line.

File: nautilus/nautilus-cast-to-tv.py [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
+from gi.repository import GObject, Gio
 gi.require_version('Nautilus', '3.0')
 gi.require_version('GObject', '2.0')
 gi.require_version('Gio', '2.0')
-from gi.repository import GObject, Gio
```
Comment (Rafostar) [inline nautilus/nautilus-cast-to-tv.py]: This is wrong. You have to specify version to import before importing.

File: nautilus/nautilus-cast-to-tv.py [hunk 3]
```diff
@@ -23,10 +28,13 @@
 _ = gettext.gettext
 
 EXTENSION_NAME = 'cast-to-tv@rafostar.github.com'
-EXTENSION_PATH = os.path.expanduser('~/.local/share/gnome-shell/extensions/' + EXTENSION_NAME)
+EXTENSION_PATH = os.path.expanduser(
+    '~/.local/share/gnome-shell/extensions/' + EXTENSION_NAME
+                                    )
```
Comment (Rafostar) [inline nautilus/nautilus-cast-to-tv.py]: Too big indent.

File: nautilus/nautilus-cast-to-tv.py [hunk 4]
```diff
@@ -69,42 +82,60 @@ def create_menu_item(self, stream_type, files):
...
+                name='CastTranscodeMenu::Video', label=_("Video")
+            )
+            sub_sub_menuitem_1.connect(
+                'activate', self.transcode_files_cb, files, stream_type, False
+            )
             submenu_2.append_item(sub_sub_menuitem_1)
 
-            #sub_sub_menuitem_2 = FileManager.MenuItem(name='CastTranscodeMenu::Audio', label=_("Audio"))
-            #sub_sub_menuitem_2.connect('activate', self.cast_files_cb, files, stream_type, True)
-            #submenu_2.append_item(sub_sub_menuitem_2)
```
Comment (Rafostar) [inline nautilus/nautilus-cast-to-tv.py]: Please do not remove commented out lines. They are there for future.

**Comment by Rafostar** [comment]
Too many changes between this and current master which make this unmergable. Since current code does not include any try/catch block that were your biggest concern. This only add cosmetic changes that are hard to merge with so many different lines in current master. Therefore closing.

**Comment by deisi** [comment]
Fine :)

Rafostar <notifications@github.com> schrieb am Mi., 8. Jan. 2020, 21:14:

**Pull request closed without merging**